docs/toml-schema.md

```diff
@@ -301,13 +301,18 @@ octocat = "write"
 # The pattern matching the branches to be protected (required)
 pattern = "master"
 # Which CI checks to are required for merging (optional)
+# Cannot be set if `pr-required` is `false`.
```

Comment:
Would it ever make sense for PRs to be optional but to require these checks _if_ a PR is opened?

Reply:
I was also thinking of this, but it's not actually possible using the GH model, AFAIK. You can enable or disable checks. Once checks are enabled, PRs are no longer optional, because without a PR you cannot do a merge.